linode/manager
# feat: [M3-6729] - Add VPC column to Linodes landing table

## Description 📝
Added vpc column to linode landing table so that users can see what vpc a linode is associated with 

## Major Changes 🔄
- add vpc column to linode landing table
- changed IP Address column to public iP address

- [ ] TODO: figure out sortable table header for vpc column ;-;

## Preview 📷
![image](https://github.com/linode/manager/assets/139280159/99dc450e-3dd4-4083-a2f5-72f15e970f70)

## How to test 🧪
1. Turn on mocks and navigate to linodes landing page to see the vpc column (you can see the vpc column without the mock as long as you have the vpc flag enabled, but you won't be able to see any mock data)
